package Protos;

import "base.proto";

message TimeSeriesData {
    repeated double values = 1;
}

message TimeSeriesDimInfo {
    required uint32 size = 1;
}

message TimeSeriesComplexData {
    repeated double values = 1;
    repeated double values_imag = 2;
}

